paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/SavedPaymentMethodTabLayoutUI.kt

```diff
@@ -345,6 +351,7 @@ private fun SavedPaymentMethodTab(
     }
 }
 
+@Suppress("MagicNumber")
```

Comment:
can you fix this instead of suppressing it? I think you just need to create a variable somewhere and use that. The downside of suppressing the error is that if we add any other "magic numbers", this error won't show up.

Reply:
Updated 👍 . It's annoying that detekt doesn't allow local variables so I made private const vals in the file

Reply:
agreed. Thanks for the update!